frontroute changed to 16.x, other versions update

vtitov opened the pull request:
Please, help to migrate the example to frontroute 16.x.
I didn't understand what to do with PopState, so I left it commented out:
https://github.com/vtitov/frontroute-example/blob/frontroute-v16/modules/frontend/src/main/scala/starter/Routes.scala#L40

yurique commented:
Hey! Thanks for taking this on! :)

`runRoute` no longer exists in `0.16.x` – no extra setup is needed. `BrowserNavigation.emitPopStateEvent()` is no longer needed either at the boot time.

vtitov commented:
> Hey! Thanks for taking this on! :)

Yes, it was quite confusing to attempt to compile code for v15 with v16 :)
And thank you for a nice tool!

> `runRoute` no longer exists in `0.16.x` – no extra setup is needed. `BrowserNavigation.emitPopStateEvent()` is no longer needed either at the boot time.

I've just removed these `runRoute` & `emitPopStateEvent`, slightly refactored `PageLayout`, added parameter for `PageNotFound`.
Please, feel free to make additional changes or reformat e.g. `Routes#route` if you think it became too compact.

yurique commented:
Hey! Sorry it's taking me longer to merge this (I want to also update some other things here and there). Will do it soon, though :)

Thanks again!
